pkg/namesgenerator/cmd/names-generator/main.go

```diff
@@ -2,10 +2,16 @@ package main
 
 import (
 	"fmt"
+	"math/rand"
+	"time"
 
 	"github.com/docker/docker/pkg/namesgenerator"
 )
 
+func init() {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
I am not a big fan of `init`. Maybe moving this into `main` before printing a random name?

Reply:
I understood. I'm gonna do it.

Reply:
I fixed it, and verified the binary action.